app/javascript/dashboard/routes/dashboard/settings/inbox/components/AddCampaign.vue

```diff
@@ -16,16 +16,16 @@
           @blur="$v.title.$touch"
         />
 
-        <label :class="{ error: $v.message.$error }">
+        <label class="editor-wrap">
           {{ $t('CAMPAIGN.ADD.FORM.MESSAGE.LABEL') }}
-          <textarea
+          <woot-message-editor
             v-model="message"
-            rows="5"
-            type="text"
+            class="editor"
+            :class="{ editor_warning: $v.message.$error }"
             :placeholder="$t('CAMPAIGN.ADD.FORM.MESSAGE.PLACEHOLDER')"
             @blur="$v.message.$touch"
           />
-          <span v-if="$v.message.$error" class="message">
+          <span v-if="$v.message.$error" class="warning-message">
```

Comment:
the `message` class is from foundation which already uses colors. Can you check if it works here ?

Reply:
I have checked it's not working here.